cpp/include/cudf/aggregation.hpp

```diff
@@ -168,6 +169,16 @@ std::unique_ptr<aggregation> make_argmin_aggregation();
 */
 std::unique_ptr<aggregation>
 make_nunique_aggregation(include_nulls _include_nulls = include_nulls::NO);
+
+/**
+ * @brief Factory to create a `nth_element` aggregation
+ * 
```

Comment:
If `n` is allowed to be negative, that _definitely_ needs to be documented.

Reply:
added documentation.